reference/features/search_parameters.md

```diff
@@ -366,3 +367,31 @@ You would get the following response with **information about the matches in the
   }
 }
 ```
+
+## Sort
+
+**Parameter**: `sort`
+**Expected value**: a list of attributes written as an array or as a comma-separated string
+**Default value**: `null`
+
+Sorts search results at query time according to the specified attributes and indicated order.
```

Comment:
It may be missing some usage information like the one you can find in [facets distribution](https://docs.meilisearch.com/reference/features/search_parameters.html#matches).

It lets the user quickly understand how the string is constructed without having to look at the examples.

```
Usage: `attribute:order`
A search formula is composed of an attribute and an order. Both are separated by a colon `:`. For example, `price:asc`. 
```

Reply:
Good for me